### PR TITLE
feat(api,cli): add organization

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17937.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17939.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

File: tests/03_clictl_admin_organization.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Database Commands TestSuite
+testcases:
+- name: Add an organization
+  steps:
+  - script: {{.cdsctl}} -f {{.cdsctl.config}} admin organization add my-org
+- name: List all organization
```
Comment (yesnault) [inline tests/03_clictl_admin_organization.yml:6]:
```suggestion
- name: List all organizations
```

File: cli/cdsctl/admin_organization.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	"context"
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+)
+
+var adminOrganizationCmd = cli.Command{
+	Name:    "organization",
+	Aliases: []string{"org", "orga"},
```
Comment (yesnault) [inline cli/cdsctl/admin_organization.go:13]:
```suggestion
	Aliases: []string{"org", "orga", "organizations"},
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17940.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17944.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17944.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17946.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✘  
  * Standalone Tests ✘

**Review by richardlt — APPROVED** [review]

**Pull request merged**